openapi/paths/reports@revenue-audit.yaml

```
@@ -0,0 +1,56 @@
+servers:
+  - url: 'https://api-sandbox.rebilly.com/experimental/organizations/{organizationId}'
+    description: Sandbox server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+  - url: 'https://api.rebilly.com/experimental/organizations/{organizationId}'
+    description: Live server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+get:
+  x-products:
+    - Reports
+  tags:
+    - Reports
+  summary: Retrieve a revenue audit report
+  operationId: GetRevenueAuditReport
+  x-sdk-operation-name: getRevenueAudit
+  description: |
+    Retrieve a revenue audit report which contains detailed journal entries.
+    A revenue audit report is a combination of a granular
+    [revenue waterfall](https://www.rebilly.com/docs/data-tables/revenue-recognition/#revenue-waterfall)
+    and a journal report, which describes revenue per account.
+    Use [`filter`](https://api-reference.rebilly.com/#section/Using-filter-with-collections),
```

nitpick: comma is not needed
```suggestion
    Use [`filter`](https://api-reference.rebilly.com/#section/Using-filter-with-collections)
```